tests/test_Producer.py

```diff
@@ -65,9 +65,9 @@ def test_produce_headers():
                   'error_cb': error_cb,
                   'default.topic.config': {'message.timeout.ms': 10}})
 
-    p.produce('mytopic', value='somedata', key='a key', headers=[('headerkey', 'headervalue')])
-    p.produce('mytopic', value='somedata', key='a key', headers=[('dupkey', 'dupvalue'), ('dupkey', 'dupvalue')])
-    p.produce('mytopic', value='somedata', key='a key', headers=[('dupkey', 'dupvalue'), ('dupkey', 'diffvalue')])
+    p.produce('mytopic', value='somedata', key='a key', headers=[('headerkey', b'headervalue')])
+    p.produce('mytopic', value='somedata', key='a key', headers=[('dupkey', b'dupvalue'), ('dupkey', b'dupvalue')])
+    p.produce('mytopic', value='somedata', key='a key', headers=[('dupkey', b'dupvalue'), ('dupkey', b'diffvalue')])
```

Comment:
We want a test that verifies that you can pass a string as value, not just binary bytes